src/inmanta/data/__init__.py

```diff
@@ -2261,6 +2275,14 @@ def to_dto(self) -> m.Environment:
             validator=convert_int,
             doc="The number of days to retain notifications for",
         ),
+        AUTO_RECOMPILE_WAIT: Setting(
```

Comment:
Not necessarily a change request, but just a proposal: Shouldn't we take this as an opportunity to improve the name of this config option? I don't like the current name for the following reasons:
* `AUTO_RECOMPILE_WAIT` sounds like a boolean option, but it isn't.
* The `AUTO` part of `AUTO_RECOMPILE_WAIT` hints to the fact that this option only applies to compiles that were triggered automatically (by the update of a fact for example). But in reality it applies on all requested compiles.

Wouldn't it be better to call this option `RECOMPILE_BACKOFF` or something similar. That clearly conveys the purpose of the setting.

Reply:
I agree, that's a better name

Reply:
@wouterdb What is your opinion on this?